pkgs/development/python-modules/objax/default.nix

```diff
@@ -12,13 +12,11 @@
 
 buildPythonPackage rec {
   pname = "objax";
-  version = "1.6.0";
+  version = "1.7.0";
 
-  src = fetchFromGitHub {
-    owner = "google";
-    repo = "objax";
-    rev = "v${version}";
-    hash = "sha256-/6tZxVDe/3C53Re14odU9VA3mKvSj9X3/xt6bHFLHwQ=";
+  src = fetchPypi {
```

Comment:
Let's file an issue about the git tag upstream and link to it in a code comment here

Reply:
Done !
Issue: https://github.com/google/objax/issues/263